external-crates/move/crates/move-compiler/src/linters/mod.rs

```diff
@@ -32,22 +39,27 @@ pub enum LinterDiagnosticCategory {
 pub const ALLOW_ATTR_CATEGORY: &str = "lint";
 pub const LINT_WARNING_PREFIX: &str = "Lint ";
 pub const CONSTANT_NAMING_FILTER_NAME: &str = "constant_naming";
-
 pub const CONSTANT_NAMING_DIAG_CODE: u8 = 1;
-
-pub enum LinterDiagCategory {
-    Style,
-}
+pub const REDUNDANT_REF_DEREF_FILTER_NAME: &str = "redundant_ref_deref";
+pub const REDUNDANT_REF_DEREF_DIAG_CODE: u8 = 2;
```

Comment:
Is it possible to automate generating these codes, similar to how we do it in `diagnostics/mod.rs`?

Reply:
It might be, but let's take that on ourselves if we want to do it